src/trpc/router/userRouter.ts

```diff
@@ -141,21 +142,34 @@ export const userRouter = router({
 
     edit: protectedProcedure.input(editUserFieldsSchema).mutation(async ({ input, ctx }) => {
         accessCheck(checkRoleForAccess(ctx.session.user.role, 'editUser'));
-        const userBefore = await userMethods.getByIdOrThrow(input.id);
+
+        const userBeforeWithCurators = await prisma.user.findUnique({
+            where: { id: input.id },
+            include: { curators: true },
+        });
+        if (!userBeforeWithCurators) throw new TRPCError({ code: 'NOT_FOUND', message: `No user with id ${input.id}` });
         const result = await userMethods.edit(input);
```

Comment:
why don't you include curators in `userMethod.edit`?

Reply:
it's really not obvious why edit method should return curators